person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, parent_permission: true, name: 'Unknown')
```

Comment:
- [ ] This parameters arrangement will bring errors, the `name` should come before the `parent_permission`:

  ```
  def initialize(age, name: 'Unknown', parent_permission: true)
  ```

Reply:
how will that bring error

Reply:
Hi @DuaneDave oh yeah. I have noticed that you used colon notation for the `name` and not an equal sign. You can ignore this requested change if you wish.

Reply:
okay, thank you 😄